packages/core/src/modules/proofs/protocol/v2/handlers/V2RequestPresentationHandler.ts

```diff
@@ -65,11 +65,8 @@ export class V2RequestPresentationHandler implements Handler {
 
     const { message, proofRecord } = await this.proofService.createPresentation({
       proofRecord: record,
-      proofFormats: {
-        indy: requestedCredentials.indy,
-      },
+      proofFormats: requestedCredentials,
       protocolVersion: ProofProtocolVersion.V2,
```

Comment:
protocolVersion is already present in the proof record, so not need to pass it to this function